### ISSUE TITLE
[Bug][6.x] Field `checklist_dependency` fails on permissions

### ISSUE THREAD

**Issue by marechenok** [Laravel-Backpack/CRUD#5249]
# Bug report

### What I did

After migration to v6 checklist_dependency field falls to 500 error page on User edit CRUD with installed backpack [PermissionManager](https://github.com/Laravel-Backpack/PermissionManager)
My code for this field:

```php
[
                // two interconnected entities
                'label' => trans('User Role Permissions'),
                'field_unique_name' => 'user_role_permission',
                'type' => 'checklist_dependency',
                'name' => 'roles, permissions',
                'subfields' => [
                    'primary' => [
                        'label' => trans('Roles'),
                        'name' => 'roles', // the method that defines the relationship in your Model
                        'entity' => 'roles', // the method that defines the relationship in your Model
                        'entity_secondary' => 'permissions', // the method that defines the relationship in your Model
                        'attribute' => 'name', // foreign key attribute that is shown to user
                        'model' => config('permission.models.role'), // foreign key model
                        'pivot' => true, // on create&update, do you need to add/delete pivot table entries?]
                        'number_columns' => 3, //can be 1,2,3,4,6
                    ],
                    'secondary' => [
                        'label' => trans('Permission'),
                        'name' => 'permissions', // the method that defines the relationship in your Model
                        'entity' => 'permissions', // the method that defines the relationship in your Model
                        'entity_primary' => 'roles', // the method that defines the relationship in your Model
                        'attribute' => 'name', // foreign key attribute that is shown to user
                        'model' => config('permission.models.permission'), // foreign key model
                        'pivot' => true, // on create&update, do you need to add/delete pivot table entries?]
                        'number_columns' => 3, //can be 1,2,3,4,6
                    ],
                ],
            ],
```
As it is in doc page.

### What I expected to happen
Expected to see roles and permissions
![image](https://github.com/Laravel-Backpack/CRUD/assets/26778514/44c9b904-610d-40b3-b0dc-376b479eaa39)

### What happened

And the error I'm getting
![image](https://github.com/Laravel-Backpack/CRUD/assets/26778514/a7ba875b-6216-4b9d-b6a1-0abc2ba1c75c)

### Is it a bug in the latest version of Backpack?

Yes, latest v6.1.1 

### Backpack, Laravel, PHP, DB version

```
### PHP VERSION:
PHP 8.1.12 (cli) (built: Oct 28 2022 17:39:37) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.12, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.16.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.0.2
backpack/crud: 6.1.1
backpack/filemanager: 3.0.2
backpack/generators: v4.0.2
backpack/newscrud: v5.0.0
backpack/pagemanager: 3.2.0
backpack/permissionmanager: 7.0.0
backpack/pro: 2.0.9
backpack/settings: 3.1.0
backpack/theme-coreuiv2: 1.1.4
```

**Comment by karandatwani92** [comment]
Hi @marechenok 
could you please share the output of `php artisan backpack:version`?
i doubt it, an older version of the permission manager is installed which is not compatible with backpack v6.

https://backpackforlaravel.com/docs/6.x/upgrade-guide#step-2
Required version:
`"backpack/permissionmanager": "^7.0",`

**Comment by marechenok** [comment]
```
### PHP VERSION:
PHP 8.1.12 (cli) (built: Oct 28 2022 17:39:37) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.12, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.18.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.1.1
backpack/crud: 6.1.7
backpack/filemanager: 3.0.3
backpack/generators: v4.0.2
backpack/newscrud: v5.0.0
backpack/pagemanager: 3.2.0
backpack/permissionmanager: 7.0.0
backpack/pro: 2.0.10
backpack/settings: 3.1.0
backpack/theme-coreuiv2: 1.1.4

```

**Comment by pxpm** [comment]
This was a tricky one. 

The problem was that @marechenok defined the field name with a space between the comma:

```diff
- 'roles,permissions'
+ 'roles, permissions'
```

I've created a PR to fix it: #5280 I am just waiting for @tabacitu spidey sense analysis before merging it. 

Thanks again for the report @marechenok and sorry it took us some time to find and fix it.

Cheers
